### PR TITLE
Fix spelling mistake

### PR DESCRIPTION
**Pull request by Joepkock02**

### CODE REVIEW

File: Extras/RealNames.cfg [hunk 1]
```diff
@@ -105,40 +105,100 @@
 
 @PART[SEP_24_HOPPY]:FOR[StarshipExpansionProject]
 {
-	@title = Starhopper
+	@title = Starhopper (Hoppy for friends :D)
```
Comment (SofieBrink) [inline Extras/RealNames.cfg:108]: Really? Lol

File: Extras/RealNames.cfg [hunk 2]
```diff
@@ -105,40 +105,100 @@
...
 }
 
 @PART[SEP_24_SHIP_PROTO_AFT_LEFT]:FOR[StarshipExpansionProject]
 {
-	@title = SN15 Rear Left Flap
+	@title = Hight-Altitude Starship Rear Left Flap
```
Comment (SofieBrink) [inline Extras/RealNames.cfg:113]: Hight? Should probably be high?
But isn’t suborbital better?

**Comment by SofieBrink** [comment]
Also targeting the wrong branch I believe

**Comment by Joepkock02** [comment]
yeah it seems like it

**Pull request closed without merging**